PyInstaller/utils/hooks/gi.py

```diff
@@ -140,9 +141,10 @@ def gir_library_path_fix(path):
                          'package.', gir_file)
             return None
 
-        with open(gir_file, 'r') as f:
+        with open_file(gir_file, text_read_mode, encoding='utf-8') as f:
             lines = f.readlines()
-        with open(os.path.join(CONF['workpath'], gir_name), 'w') as f:
+        with open_file(os.path.join(CONF['workpath'], gir_name), 'w',
+                  encoding='utf-8') as f:
```

Comment:
How can we be sure utf-8 is expected or accepted? If you have some source for this assumption, please state this in a comment.

Reply:
As before, let's specify and encoding, instead of relying on a poorly-documented "default".